src/stores/Content.jsx

```diff
@@ -16,6 +16,7 @@ import contentTerms from "../assets/json/terms.json";
 export default class ContentStore {
   @observable content = { faq: {}, tooltips: {} }
   @observable contentLoaded = false
+  @observable showNotification = true;
```

Comment:
hard to know without testing but can this maybe not be 

```js
@observable showNotication = !localStorage.getItem(`StabilityFeeChangeAlertClosed-${this.stabilityFeeMarkdown()}`)
```
?

Reply:
I tried this and it runs into issues until content is loaded. Moved to the `then()` of the axios request.